src/utils/track.ts

```diff
@@ -159,6 +159,19 @@ export const trackFeedbackSubmission = (feedback: boolean) => {
     }
   }
 
+  AWSMA.ready(() => {
+    document.dispatchEvent(new CustomEvent(AWSMA.TRIGGER_EVENT, {detail: opt}));
+  });
+}
+
+export const trackCopyClicks = () => {
+  const opt = {
+    event: {
+      type: 'click',
+      name: 'copied'
```

Comment:
Should we also track what was copied like how Amplify UI does it? https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/utils/track.ts#L204-L206

Reply:
agreed, that has been added